File: app/src/main/java/com/example/gastosinteligentes/database/entidades/Usuario.java
package com.example.gastosinteligentes.database.entidades;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "usuarios")
public class Usuario {

    @PrimaryKey(autoGenerate = true)
    public int id;

    public String correo;
    public String contraseña;
    public String nombre;
}

File: app/src/main/java/com/example/gastosinteligentes/database/entidades/Usuario.kt
package com.example.gastosinteligentes.database.entidades

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "usuarios")
data class Usuario(

    @PrimaryKey(autoGenerate = true)
    var id: Int = 0,

    var correo: String = "",

    var contraseña: String = "",

    var nombre: String = ""
)